core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -100,11 +100,11 @@ static contextEndFunction contextEndCallee = nullptr;
 
 namespace Profiling {
 
-bool profileLibraryLoaded() { return (nullptr != initProfileLibrary); }
```

Comment:
This commit (b6cf9cba2c2f92a658956363dd7ca7987725ad48) would likely be a good candidate for a pull request on its own. :wink:

Reply:
@masterleinad I'm hoping that if I do enough offensive things in one PR, a few will slip under the radar ;)